provider/keycloak_openid_client_optional_scopes_test.go

```diff
@@ -426,6 +424,7 @@ resource "keycloak_openid_client_optional_scopes" "optional_scopes" {
 		"address",
 		"phone",
 		"offline_access",
+		"microprofile-jwt",
```

Comment:
I assume this optional scope is now added to every client by default as of 5.0.0 or 6.0.0?  I'm still on 4.8.3.Final.